src/queries/group_query.cr

```diff
@@ -1,2 +1,5 @@
 class GroupQuery < Group::BaseQuery
+  def contains_user?(group_id : Int64, user : User)
+    id.eq(group_id).where_users(UserQuery.new.id.eq(user.id)).any?
+  end
```

Comment:
This might be simpler on the MembershipQuery

Something like: MembershipQuery.new.user_id(user.id).group_id(group_id).first?

Up to you though! Just an idea

Reply:
Oh yeah I like that. I got hung up thinking about the verbage of a `Group` containing a `User`.